ts/src/bitget.ts

```diff
@@ -2251,6 +2252,12 @@ export default class bitget extends Exchange {
         if (since !== undefined) {
             request['startTime'] = since;
         }
+        const until = this.safeInteger2 (params, 'until', 'endTime');
```

Comment:
@Dan-krm u sure this is the case for every endpoint? What if the user provides the `since` only? are we required to provide an `endTime` in the request? If so we should probably default it to `this.milliseconds()` (now)

Reply:
Yeah I checked it with every endpoint, and yeah sure defaulting endTime to this.milliseconds might be useful in some cases